### PR TITLE
Better Linux instructions for downloading

### PR DESCRIPTION
**Pull request by LimesKey**
I have added better Linux instructions for downloading Java and setting the JAVA_HOME path variable on Linux. If anyone has any other suggestions to add or change, please let me know as I'm not that great at Linux. I attempted this guide to downloading on a Linux Codespace VM and it worked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,18 +135,47 @@ The results of your analysis will be shown in the right pane.
...
-The easiest way to run the program on Linux or macOS is by using the executable file located in the [releases](https://github.com/VerisimilitudeX/DNAnalyzer/releases/latest).
 
-First, download the zip or tar.gz file from releases and unzip it using the native utility. Then build gradle and run the GUI.
 
-```pwsh
+### DNAnalyzer & Java Download
+
+To run DNAnaylzer on Linux, you'll need to download the DNAnalyzer source code and download and install Java 17.
+
+First, download the zip or tar.gz file from [DNAnalyzer releases](https://github.com/VerisimilitudeX/DNAnalyzer/releases/latest) and unzip it using the native utility.
```
Comment (VerisimilitudeX) [inline README.md:144]: Your native utility

File: README.md [hunk 2]
```diff
@@ -135,18 +135,47 @@ The results of your analysis will be shown in the right pane.
...
-First, download the zip or tar.gz file from releases and unzip it using the native utility. Then build gradle and run the GUI.
 
-```pwsh
+### DNAnalyzer & Java Download
+
+To run DNAnaylzer on Linux, you'll need to download the DNAnalyzer source code and download and install Java 17.
+
+First, download the zip or tar.gz file from [DNAnalyzer releases](https://github.com/VerisimilitudeX/DNAnalyzer/releases/latest) and unzip it using the native utility. 
+
+Then, download [Java 17](https://www.oracle.com/ca-en/java/technologies/downloads/#java17), find the download at https://www.oracle.com/ca-en/java/technologies/downloads/#java17, ensure you choose the Linux option and get the one for your correct processor architecture.
```
Comment (VerisimilitudeX) [inline README.md:146]: And ensure

**Review by VerisimilitudeX — CHANGES_REQUESTED** [review]
Some minor issues - I'll fix them in a couple hours.

File: README.md (hunk 1 above)
Comment (LimesKey) [reply README.md:144]: yeah, like whatever zip manager you have, 7zip, gzip, tar

File: README.md (hunk 2 above)
Comment (LimesKey) [reply README.md:146]: i see nothing wrong with that

**Review by VerisimilitudeX — APPROVED** [review]

**Pull request merged**